.pre-commit/.just

```diff
@@ -40,6 +40,12 @@ check-container-runtime:
         [podman]="podman info > /dev/null 2>&1"
     )
 
+    # If DOCKER_HOST is set print it.
+    # Sometimes it has the wrong value, so it's good to know.
+    if [[ -n "${DOCKER_HOST:-}" ]]; then
+        echo "DOCKER_HOST=$DOCKER_HOST"
+    fi
+
```

Comment:
Not relevant to this PR, just a god opportunity to sneak this in.

Based on a true story -- I had my DOCKER_HOST set incorrectly, basically forgotten from the https://github.com/Talus-Network/nexus-sdk/issues/262 investigation.